src/tools/rbd_mirror/ImageMap.h

```diff
@@ -156,11 +156,11 @@ class ImageMap {
   void schedule_rebalance_task();
 
   void notify_listener_acquire_release_images(const Updates &acquire, const Updates &release);
-  void notify_listener_remove_images(const std::string &peer_uuid, const Updates &remove);
+  void notify_listener_remove_images(const std::string &mirror_uuid, const Updates &remove);
```

Comment:
If touching this line, let's make it right :-)
```suggestion
  void notify_listener_remove_images(const std::string &mirror_uuid,
                                     const Updates &remove);
```

Reply:
Done